docs/decisions/0007-compile-common-theme-for-all-mfes.rst

```diff
@@ -0,0 +1,139 @@
+0007 Compile Common Theme for all MFEs
+######################################
+
+
+Status
+******
+
+Draft
+
+Context
+*******
+
+Currently each Open edX MFE includes a small snippet of SCSS code that is common
+to all of them, which is the following four lines:
+
+.. code-block:: SCSS
+
+	@import "~@edx/brand/paragon/fonts";
+	@import "~@edx/brand/paragon/variables";
+	@import "~@edx/paragon/scss/core/core";
+	@import "~@edx/brand/paragon/overrides";
+
+These four lines, import the Paragon and bootstrap code and apply
```

Comment:
Hi @xitij2000, I think this is a good idea with the actual status of the theming. 

I was wondering about the header, footer, and cookie banner styles, that should be updated to use CSS variables, update the version in the MFE and be part of the MFE stylesheet or could be part of the theme stylesheet?

Reply:
I think the header, footer, cookie banner and other elements shoudl also over time be updated to use CSS variables. They can then be independent stylesheets that are loaded for those. 

So we'd have:

1. Base theme stylesheet (loaded from a common source)
2. Component stylesheets (header, footer, banner etc. separate for each MFE)
3. MFE stylesheet (separate for each MFE).